packages/cardano-services/test/Program/services/ogmios.test.ts

```diff
@@ -400,6 +404,36 @@ describe('Service dependency abstractions', () => {
       await provider.shutdown();
     });
 
+    it('should execute a provider error from the ogmios, and be intercepted by txSubmit', async () => {
```

Comment:
```suggestion
    it('throws a provider error if the submitted transaction does not contain addresses that can be resolved from the included context', async () => {
```
Is this it?

Reply:
It is

Reply:
aa8096909c8ee4b312f8d2d9395b28766ebd4ff2 should solve this